api/src/main/java/org/openmrs/module/pihmalawi/activator/AuthenticationInitializer.java

```diff
@@ -0,0 +1,135 @@
+package org.openmrs.module.pihmalawi.activator;
```

Comment:
This follows the same pattern that we have in the PIH EMR, and sets up the necessary authentication schemes and configuration programmatically so that it can be done consistently across environments without needing specific runtime configurations to be set.